### DISCUSSION TITLE
New Install only shows report menu?

### DISCUSSION THREAD

**Discussion by PDXDave23** [opendcim/openDCIM#1453] [Q&A]
Sounds like this issue may have been answered on the old mailing list. I can't seem to find anything to address this issue now. When I login the only thing I have access to is the menu. Anyone offer any pointers? Thanks.

<img width="1078" alt="image" src="https://github.com/opendcim/openDCIM/assets/147285592/a57280cc-4f65-49e0-a982-dd84090206de">

**Comment by wilpig** [comment]
https://github.com/opendcim/openDCIM/issues/1452

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: PDXDave23 ***@***.***>
Sent: Saturday, October 7, 2023 6:51:55 PM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [opendcim/openDCIM] New Install only shows report menu? (Discussion #1453)


Sounds like this issue may have been answered on the old mailing list. I can't seem to find anything to address this issue now. When I login the only thing I have access to is the menu. Anyone offer any pointers? Thanks.

[image]<https://user-images.githubusercontent.com/147285592/273417762-a57280cc-4f65-49e0-a982-dd84090206de.png>

—
Reply to this email directly, view it on GitHub<https://github.com/opendcim/openDCIM/discussions/1453>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAMTY6WPJLPH6MUWCNHGJPLX6HTJXAVCNFSM6AAAAAA5XH6ZBWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVG4YTCNZSHE>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/opendcim/openDCIM/discussions/1453", "url": "https://github.com/opendcim/openDCIM/discussions/1453", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by cyj2310256** [reply]
according to #1452 , it's dcim account permission issue, i recreate a new admin and grant all permissions, then change the admin account in "db.inc.php", but still same issue.  I plan to install another verion, someone know which previous verion works well ?

**Reply by wilpig** [reply]
There is no admin account set in db.inc.php

**Reply by cyj2310256** [reply]
so, still permission issue ? 
![2023-11-16 09_25_52](https://github.com/opendcim/openDCIM/assets/89081931/faf73e37-3bcf-4d45-8d9a-e66f97b2a736)
I paste dcim account permission, any problem ?

**Comment by samilliken** [comment]
You are likely not passing the user info into openDCIM.   What username shows in the top right under the language selection?

 

Scott

**Comment by wilpig** [comment]
Just stop messing with 23.03 and wipe it and use 23.04 and be done with it.  The bug was fixed, follow the install doc int he wiki and win.

https://github.com/opendcim/openDCIM/wiki/RHEL
https://github.com/opendcim/openDCIM/wiki/Ubuntu
https://github.com/opendcim/openDCIM/wiki/OpenSUSE

**Reply by cyj2310256** [reply]
I rebuild the environment with this latest guide, it works now, thanks a lot. the next step is learn to use this tool
